<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>593</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="password_length_slider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password Generator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="slider_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password Length: 0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="include_numbers_checkbox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Include Numbers</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="include_specialChars_checkbox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Include Special Characters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Generate Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copy_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Copy Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="copy_success_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>520</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="password_field">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>426</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
